### PR TITLE
Add additional rules to support system.web.mvc

### PR DESCRIPTION
**Pull request by marknfawaz**

### CODE REVIEW

File: recommendation/system.web.mvc.json [hunk 1]
```diff
@@ -269,14 +303,427 @@
...
+            {
+              "Name": "ReplaceMethodOnly",
+              "Type": "Method",
+              "Value": {
+                "oldMethod": "HttpNotFound",
+                "newMethod": "NotFound"
+              },
+              "Description": "Replace HttpNotFound with NotFound",
+              "ActionValidation": {
+                "NotContains": "HttpNotFound"
```
Comment (Eruanion) [inline recommendation/system.web.mvc.json:356]: Should we have a contains here just for consistency?

File: recommendation/system.web.mvc.json [hunk 2]
```diff
@@ -269,14 +303,427 @@
...
+            {
+              "Name": "ReplaceMethodOnly",
+              "Type": "Method",
+              "Value": {
+                "oldMethod": "TryUpdateModel",
+                "newMethod": "TryUpdateModelAsync"
+              },
+              "Description": "Replace TryUpdateModel with TryUpdateModelAsync",
+              "ActionValidation": {
+                "Contains": "TryUpdateModelAsync"
```
Comment (Eruanion) [inline recommendation/system.web.mvc.json:625]: Same here for consistency should we add a notcontains?

**Review by Eruanion — COMMENTED** [review]
Just added a couple of comments for consistency looks good.

File: recommendation/system.web.mvc.json (hunk 1 above)
Comment (marknfawaz) [reply recommendation/system.web.mvc.json:357]: Sure, updated.

File: recommendation/system.web.mvc.json (hunk 2 above)
Comment (marknfawaz) [reply recommendation/system.web.mvc.json]: Sure, updated.

**Review by Eruanion — APPROVED** [review]

**Pull request merged**